lib/posix/spawn/child.rb

```diff
@@ -148,8 +154,9 @@ def exec!
       rescue Object => boom
         [stdin, stdout, stderr].each { |fd| fd.close rescue nil }
         if @status.nil?
-          ::Process.kill('TERM', pid) rescue nil
-          @status = waitpid(pid)      rescue nil
+          kill_pid = @pgroup_kill ? pid * -1 : pid
```

Comment:
> Weird, seems to work w/ either a negative pid or "-TERM" signal string. I'll go with the documented approach here.
Add a line note

Looks like if you [pass a negative signal](https://github.com/github/ruby/blob/2.1/signal.c#L438-L444), it just [negates the pid](https://github.com/github/ruby/blob/2.1/signal.c#L388), which is what means "kill the group" in the kill syscall. Dunno why *that* isn't documented.